game_event/tests.py

```diff
@@ -179,3 +198,86 @@ def test_post_game_event(self, client, saved_court, court_ball_game):
         assert GameEvent.objects.count() == count_of_game_events + 1
         assert response.status_code == 302
         assert response.url == '/game-events/'
+
+
+@pytest.mark.django_db
+class TestGameEventViews:
+
+    def test_success_loading_site_game_event(self, client, saved_game_event, player):
+        client.force_login(player.user)
+        game_event_id = saved_game_event.id
+        url = reverse('game_event', args=[game_event_id])
+        response = client.get(url)
+        assert response.status_code == 200
+
+    def test_fail_loading_site_game_event(self, client, player):
```

Comment:
If the game event id doesn't exist

Reply:
> Is there a scenario in which there will be a failure in loading site game event, without getting response status code 404 ? @amitKaplansky @MaayanMashhadi

I think no, because if you look at the view function (game_event), you will see
that if there isn't game_event with specific id mentioned (on this test: -1) the response status code will be 404.
`event = get_object_or_404(GameEvent, pk=id)`

Reply:
@MaayanMashhadi @amitKaplansky Oh I see, I thought about the possibility will get response status code 200 that everything went well with the response, but logic-wise, there were other problems such as if the game event id doesn't exist, and so the request won't go well even though got status code 200. but considering what maayan stated, it seems fine.